docs/root/configuration/http_filters/lua_filter.rst

```diff
@@ -290,16 +290,16 @@ under the filter name i.e. *envoy.lua*. Below is an example of a *metadata* in a
 
 Returns a :ref:`metadata object <config_http_filters_lua_metadata_wrapper>`.
 
-requestInfo()
+streamInfo()
```

Comment:
I didn't realize that we changed the Lua API. This is might be a change that we need to revert if people complain. In general I don't think we can make breaking changes like this but this might be OK since I'm guessing very few people use this. We will see.

Reply:
Oops, yeah. I think @ramaraochavali has an idea of it?

Reply:
We are not using this right now. So good from our side.

Reply:
Thanks, @ramaraochavali. @yskopets I hope you're OK with this too.